.rubocop.yml

```diff
@@ -1,11 +1,13 @@
-inherit_from:
-  - http://shopify.github.io/ruby-style-guide/rubocop.yml
+inherit_from: .rubocop_todo.yml
+
+inherit_gem:
+  rubocop-shopify: rubocop.yml
 
 AllCops:
   Exclude:
     - 'vendor/**/*'
     - 'tmp/**/*'
-  TargetRubyVersion: '2.4'
+  TargetRubyVersion: 2.5
```

Comment:
Hum, we still support Ruby 2.3

Reply:
We do, but since it is EOL and parser doesn't include support for it anymore we should probably up the min ruby version if it isn't too much trouble.

Reply:
I'd rather drop the dependency on the company style guide and use an older parser/rubocop.